### ISSUE TITLE
Sorry to contact you this way, but I could not find any other way to do it.

### ISSUE THREAD

**Issue by oleoj** [DccPlusPlus/Documentation#1]
Hi,

I just found the DCC++ library on the internet today and I tried to join TrainBoard. I could not find any way to contact them. I am living in Norway and they ask for something that is common at the back on older US trains. I have tried some of them like handwheel, lantern, sign and so on. But with no luck. 
Your work looks very promising and I your design goals are very good. I am a software developer with some knowledge of hardware.

Kind Regards
oleoj

**Comment by DccPlusPlus** [comment]
Hi Oleoj,

Thanks for contacting me and I'm sorry to hear about the problem joining TrainBoard.  When I joined about 6 years ago I don't think there was a question, but I emailed the administrator of the site and let him know the question was apparently causing some confusion.  Just so you know, yours is actually the second message on this that I have received.  When the TrainBoard admin gave me the question I immediately understood the source of the confusion, and suggested that the question be rephrased to something like "What is the common name for the trailing train car that was often found at the end of many older trains."  Of course this assumes that I'm correct in what I think the answer is :).  Let me know if this works.
